### ISSUE TITLE
EncryptionUtil errors

### ISSUE THREAD

**Issue by Innectic** [NukkitX/Protocol#25]
It seems that none of the methods in EncryptionUtils work, due to the following exception being thrown:

```
Dec 05, 2019 1:15:47 AM com.nukkitx.network.raknet.RakNetServer$ServerDatagramHandler exceptionCaught
SEVERE: An exception occurred in RakNet
java.lang.ExceptionInInitializerError
com.nukkitx.protocol.bedrock.BedrockRakNetSessionListener.onEncapsulated(BedrockRakNetSessionListener.java:28)
        at com.nukkitx.network.raknet.RakNetSession.onEncapsulatedInternal(RakNetSession.java:301)
        at com.nukkitx.network.raknet.RakNetSession.onOrderedReceived(RakNetSession.java:428)
        at com.nukkitx.network.raknet.RakNetSession.checkForOrdered(RakNetSession.java:406)
        at com.nukkitx.network.raknet.RakNetSession.onRakNetDatagram(RakNetSession.java:394)
        at com.nukkitx.network.raknet.RakNetSession.onDatagram(RakNetSession.java:273)
        at com.nukkitx.network.raknet.RakNetServer$ServerDatagramHandler.channelRead(RakNetServer.java:288)
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362)
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348)
        at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:340)
        at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1434)
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362)
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348)
        at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:965)
        at io.netty.channel.nio.AbstractNioMessageChannel$NioMessageUnsafe.read(AbstractNioMessageChannel.java:93)
        at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:647)
        at io.netty.channel.nio.NioEventLoop.processSelectedKeysPlain(NioEventLoop.java:547)
        at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:501)
        at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:461)
        at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:884)
        at java.base/java.lang.Thread.run(Unknown Source)
Caused by: java.lang.NullPointerException
        at com.nukkitx.protocol.bedrock.util.EncryptionUtils.<clinit>(EncryptionUtils.java:43)
        ... 26 more
```

This particular one is from `verifyJwt`, but it seems to happen on all methods.

**Comment by SupremeMortal** [comment]
What is your java version and library version?

**Comment by Innectic** [comment]
I tried this on both 10, and 8 (201). Currently using protocol v388 on 2.4.1

**Comment by SupremeMortal** [comment]
Fixed by 2b18ee6ce7882e3f305b588d35d227f1d5d8395e
Release 2.4.2
